cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/DistributedProgramRuntimeService.java

```diff
@@ -348,7 +348,9 @@ public ProgramLiveInfo getLiveInfo(Id.Program program, ProgramType type) {
 
     public ClusterResourceReporter(MetricsCollectionService metricsCollectionService, Configuration hConf,
                                    CConfiguration cConf) {
-      super(metricsCollectionService.getCollector(ImmutableMap.of(Constants.Metrics.Tag.CLUSTER_METRICS, "true")));
+      super(metricsCollectionService.getCollector(
+        ImmutableMap.of(Constants.Metrics.Tag.NAMESPACE, Constants.SYSTEM_NAMESPACE,
```

Comment:
this is not what we talked about, isn't it? We said we will not add system tag explicitly, and on the processing side - if namespace tag is not there, we'll assume it is "system". Do you think we should do it differnetly?